Snowflake.Data/Core/SFReusableChunk.cs

```diff
@@ -66,21 +72,25 @@ private class BlockResultData
             private static int metaBlockLengthBits = 15;
             private static int metaBlockLength = 1 << metaBlockLengthBits;
 
-            private readonly List<char[]> data = new List<char[]>();
+            private readonly List<byte[]> data = new List<byte[]>();
             private readonly List<int[]> offsets = new List<int[]>();
             private readonly List<int[]> lengths = new List<int[]>();
             private int nextIndex = 0;
             private int currentDatOffset = 0;
 
+            int savedRowCount;

```

Comment:
I used them for debugging an issue I found while testing. They have no function now. They can be deleted if they bother you.

The issue was that the chunk downloader does not work properly with prefetchslot = 1 if chunkInfos.Count > 1.

Since it is possible for the user to change the setting of prefetchthreads manually, this issue might be more important than I originally thought.

Reply:
I actually made a change in the setting of prefetchSlot to disallow use of a single prefetch slot since this does not work.